packages/payment-processor/src/payment/utils-near.ts

```diff
@@ -54,7 +54,6 @@ export async function processNearPayment(
     await contract.transfer_with_reference(
```

Comment:
Pending on @BeskydAbc review

Reply:
New format to call the contract method:
pub fn transfer_with_reference(
        &mut self,
        to: ValidAccountId,
        payment_reference: String
    ) -> Promise
[https://github.com/near-request/request-network/commit/c03933c6eea0c7bcb66d325d311f26feb3052c6b#diff-e0304f6bdeb635ddea4f7356d7aa751c27aaed77fa98f3e79ab6346ae14082f6]